app/models/video.py

```diff
@@ -0,0 +1,61 @@
+from app import db
+
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    release_date = db.Column(db.DateTime, nullable=False)
+    total_inventory = db.Column(db.Integer, nullable=False)
+    available_inventory = db.Column(db.Integer, nullable=False)
```

Comment:
Like with `Customer`, we can compute `available_inventory` with a relationship:
```rentals = db.relationship('Rental', back_populates='video', lazy=True)```

Then in `to_dict`,  the `available_inventory` isthe total inventory minus length of the `rentals`.
```available_inventory": self.total_inventory - len(self.rentals)```